.eslintrc.json

```diff
@@ -20,53 +20,12 @@
         ],
         "consistent-return": 0,
         "no-console": "off",
-        "no-unused-vars": "off",
```

Comment:
why are you deleting these eslint rules?

Reply:
check all rules in red.

Reply:
Hello @T2Wil , I did not delete anything I just accepted new changes and the guys which have been deleted, it is because I accepted the incoming changes from develop.